chapter_15_advanced_forms.asciidoc

```diff
@@ -795,32 +806,37 @@ use Django instead, it seems like we need to bring back our custom
 ====
 [source,diff]
 ----
-@@ -13,13 -14,26 +14,26 @@@
-  <div class="col-lg-6 text-center">
-      <h1 class="display-1 mb-4">{% block header_text %}{% endblock %}</h1>
--     <form method="POST" action="{% block form_action %}{% endblock %}">
--         {{ form.text }}
-+     <form
-+         class="was-validated"  <1>
-+         method="POST"
-+         action="{% block form_action %}{% endblock %}"
-+     >
-          {% csrf_token %}
-+         <input
-+             id="id_text"
-+             name="text"
-+             class="form-control form-control-lg {% if form.errors %}is-invalid{% endif %}"  <2>
-+             placeholder="Enter a to-do item"
-+             value="{{ form.text.value }}"
-+             aria-describedby="id_text_feedback"
-+             required
-+         />
-          {% if form.errors %}
--             <div class="invalid-feedback">{{ form.errors.text }}</div>
-+             <div class="invalid-feedback">{{ form.errors.text.0 }}</div>  <3>
-          {% endif %}
-      </form>
-  </div>
+@@ -14,12 +14,25 @@
+       <div class="row justify-content-center p-5 bg-body-tertiary rounded-3">
+         <div class="col-lg-6 text-center">
+           <h1 class="display-1 mb-4">{% block header_text %}{% endblock %}</h1>
+-
+-          <form method="POST" action="{% block form_action %}{% endblock %}" >
+-            {{ form.text }}
+-            {% csrf_token %}
++          <form
++              class="was-validated"
++              method="POST"
++              action="{% block form_action %}{% endblock %}"
++          >
++              {% csrf_token %}
++              <input
++                  id="id_text"
++                  name="text"
++                  class="form-control
++                         form-control-lg
++                         {% if form.errors %}is-invalid{% endif %}"
++                  placeholder="Enter a to-do item"
++                  value="{{ form.text.value }}"
++                  aria-describedby="id_text_feedback"
++                  required
++              />
+             {% if form.errors %}
+-              <div class="invalid-feedback">{{ form.errors.text }}</div>
++              <div class="invalid-feedback">{{ form.errors.text.0 }}</div>
+             {% endif %}
+           </form>
+         </div>
 ----
 ====
 
```

Comment:
TODO 
Put the <1>, <2>, <3> marks back or find another way to point out the specific changes